e2e/README.md

```diff
@@ -132,6 +134,35 @@ npm install
 npm run test:e2e:updatesnapshots
 ```
 
+## Accessibility (a11y) Testing
+
+Accessibility testing in Open MCT is addressed and enforced in two ways:
+1. Open MCT prefers Playwright's locator strategy for [page.getByRole('')](https://playwright.dev/docs/api/class-framelocator#frame-locator-get-by-role). This enforces that the elements on the page are accessible with assistive technolgies but does not enforce conformance to the a11y guidelines (handled in #2)
+2. Open MCT checks for a11y violations and enforces conformance with the [playwright axe plugin](https://playwright.dev/docs/accessibility-testing). The a11y checks are performed in the `afterEach()` of the visual tests. This allows us to leverage the coverage provided visual tests and also specifically address `color-contrast` violations which are specifically exposed in the nature of visual testing.
+
+### a11y Standards (WCAG and Section 508)
+
+Playwright axe supports a wide range of [WCAG Standards](https://playwright.dev/docs/accessibility-testing#scanning-for-wcag-violations) to test against. Open MCT is testing against the [Section 508](https://www.section508.gov/test/testing-overview/) accessibility guidelines with the intent to support higher standards over time. As of 2024, Section508 requirements now map completely to WCAG 2.0 AA. In the future, Section 508 requirements may map to WCAG 2.1 AA.
+
+### Reading a a11y test failure
+
+When an a11y test fails, the result must be interpreted in the html test report or the a11y report json artifact stored in the `/test-results/` folder. The json structure should be parsed for `"violations"` by `"id"` and identified `"target"`. Example provided for the 'color-contrast-enhanced' violation.
+
+```json
+  "violations": [
+    ...
+    {
+      "id": "color-contrast-enhanced",
+      ...
+      "impact": "serious",
+      "html": "<span class=\"label c-indicator__label\">0 Snapshots <button aria-label=\"Show Snapshots\">Show</button></span>",
+        "target": [
+          ".s-status-off > .label.c-indicator__label"
+        ],
+        "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 6.51 (foreground color: #aaaaaa, background color: #262626, font size: 8.1pt (10.8px), font weight: normal). Expected contrast ratio of 7:1"
+      },
+```
+
 ## Performance Testing
```

Comment:
It looks like the wrong cr is being applied here, but i don't see anything in vscode